recipes/database.rb

```diff
@@ -29,17 +29,16 @@
 
 db = node['wordpress']['db']
 
-if is_local_host? db['host']
+#if is_local_host? db['host']
   include_recipe "mysql::server"
 
   mysql_bin = (platform_family? 'windows') ? 'mysql.exe' : 'mysql'
-  user = "'#{db['user']}'@'#{db['host']}'"
+  user = "'#{db['user']}'@'%'"
```

Comment:
ideally would be to give access only for those hosts, that need that access (app servers)